docs/rest-api.md

```diff
@@ -322,6 +323,70 @@ whitelist
 
 ```
 
+### Message WebSocket
+
+The API Server makes available a websocket endpoint for subscribing to RPC messages
+from the FreqTrade Bot. This can be used to consume real-time data from your bot, such as entry/exit fill messages, whitelist changes, populated indicators for pairs, and more.
+
+Assuming your rest API is set to `127.0.0.1` on port `8080`, the endpoint is available at `http://localhost:8080/api/v1/message/ws`.
+
+To access the websocket endpoint, the `ws_token` is required as a query parameter in the endpoint URL. This is set in your rest API config section. To generate a safe `ws_token` you can run the following code:
+
+``` python
+>>> import secrets
+>>> secrets.token_urlsafe(16)
+'zs9XYCbTPKvux46UJckflw'
```

Comment:
can we recommend to use a higher number here (like 25 or so)? 

the result will anyway not be a readable pass - so we assume it's copy/pasted anyway.